cmd/internal/shared/joinserver/config.go

```diff
@@ -24,4 +24,5 @@ var DefaultJoinServerConfig = joinserver.Config{
 	JoinEUIPrefixes: []types.EUI64Prefix{
 		{},
 	},
+	DefaultJoinEUI: types.EUI64{0x70, 0xb3, 0xd5, 0x7e, 0xd0, 0x00, 0x00, 0x00},
```

Comment:
This is not a good default value.

If we don't support not having a default value, it should be zero.

This value is reserved for The Things Join Server.

Reply:
I see. Ok that's fine with me. I've updated to just use zero.